End-to-End-Tests.md

```diff
@@ -1,5 +1,6 @@
 ## Table of Contents
 
+* [Table of Contents](#table-of-contents)
```

Comment:
is this expected? (We already have one on the top, right?)

Reply:
fixed